cypress/support/commands.ts

```diff
@@ -37,3 +37,15 @@ Cypress.Commands.add(
     cy.visit(`/resource/${resourceId}?${Cypress.env('auth-query-parameters')}`);
   }
 );
+
+Cypress.on('uncaught:exception', (err: Error): boolean | Error => {
+  if (
+    /ResizeObserver loop completed with undelivered notifications/.test(
```

Comment:
This kind of feels like a hack, hmm... Do I understand this correct: the new hds library (I assume this error has something to do with them?) will throw an error with this message during our cypress tests? Does this happen also during our normal application usage? Should this be something we should report back to them?

Reply:
Yes, and it is the new languageSelector what is causing this

Reply:
I commented this in the HDS Slack

Reply:
https://helsinkisolutionoffice.atlassian.net/browse/HDS-526

Reply:
And no, it does not affect normal application usage. There is no errors in console or anything. It happens when running the Cypress tests. The tests will fail without ignoring the exception.